invenio_app_ils/ui/src/invenio_app_ils/common/components/Notifications/messages.js

```diff
@@ -1,7 +1,6 @@
 import React, { Component } from 'react';
 import { Message } from 'semantic-ui-react';
-
-const SUCCESS_AUTO_DISMISS_SECONDS = 60 * 10;
+import { default as config } from './config';
```

Comment:
Export it directly like the rest of the properties from the config
`import { SUCCESS_AUTO_DISMISS_SECONDS } from './wherever_our_config_is'`